src/Combiner.cpp

```diff
@@ -264,13 +268,13 @@ void CombinerInfo::setCombine(u64 _mux )
 	const u64 key = getCombinerKey(_mux);
 	if (m_pCurrent != nullptr && m_pCurrent->getKey() == key) {
 		m_bChanged = false;
-		m_pCurrent->update(false);
+		m_pCurrent->update(true);
```

Comment:
It is bad approach to mask problem instead of investigation, what is wrong.

If force update really helps, we must know, why.